src/components/shared/SearchInput.css

```diff
@@ -58,6 +58,7 @@
   flex-grow: 1;
   height: 24px;
   min-width: 40px;
+  width: 40px;
```

Comment:
Functionally this works great but could you explain how `width` is stretching?  Because of the `flex-grow`/`align-self`?

Should we remove the `min-width`?

Awesome work @th7nder~

Reply:
That's a good question. 
Inputs have **a default width,** which is set as a `flex-basis`. I thought, that setting `flex-grow: 1` to allow growth is enough, because `flex-shrink: 1` is set by default and the item will be responsive. 
I have experimented a bit and it turns out that the default `flex-shrink` _does not work on inputs_, thus allowing us only to **grow** an input's width.
This malfunction forces us to set `width` to some minimum value.

`align-self` property affects only the cross axis, it stretches an item vertically.
I think you are right, `min-width` is not necessary there.